web/assets/main.scss

```
@@ -38,6 +38,21 @@ $govuk-assets-path: "/lpa/frontend/assets/";
   border-bottom: none;
 }
 
+.app-\!-caseworker-summary {
+  background-color: #f2f6fc;
+  padding: 2px govuk-spacing(4) 4px govuk-spacing(4); // the 2px top 4px bottom is so that on click/hover of the show/hide button, the box shadow doesnt exceed the height of the case summary when collapsed
+  margin: 0 govuk-spacing(-3) govuk-spacing(2) govuk-spacing(-3);
+}
+
+.app-\!-svg-icon {
+  margin-right: govuk-spacing(1);
+}
+
+.app-\!-table-head__no-vertical-padding th {
```

This is really picky, but if you're doing BEM then you should use `--` for modifiers and `__` for children. e.g. `.button__icon--selected`

```suggestion
.app-\!-table-head--no-vertical-padding th {
```